packet_flight/viewer.html

```diff
@@ -1,25 +1,38 @@
-<html lang="en"><head>
-<meta charset="utf-8">
-<title>Packet Flight</title>
+<html lang="en">
 
-<script src="js/jquery.min.js"></script>
-<script src="js/raphael-min.js"></script>
-<script src="js/packet-flight.js"></script>
-<script src="js/data.js"></script>
+<head>
```

Comment:
Missing `</head>`

Reply:
did not change what was there in the original, but yeah makes sense...